java/idl2jni/runtime/idl2jni_runtime.h

```diff
@@ -89,7 +89,7 @@ void copyToCxx(JNIEnv *jni,
 template <typename Traits>
 void copyToJava(JNIEnv *jni, jobject &target,
   const TAO::details::object_reference_const_sequence_element<Traits> &source,
-  bool createNewObject = false)
+  bool /* createNewObject = false */)
```

Comment:
Based on my search. copyToJava() are generated in *SupportJC.cpp file for some custom types and some of the generated function use the 3rd parameter. The one I changed is just an overloaded version for some unknown types.


Huang-Ming


> On Sep 9, 2016, at 1:40 PM, Adam Mitz <notifications@github.com> wrote:
> 
> In java/idl2jni/runtime/idl2jni_runtime.h <https://github.com/objectcomputing/OpenDDS/pull/300#discussion_r78228885>:
> 
> > @@ -89,7 +89,7 @@ void copyToCxx(JNIEnv *jni,
> >  template <typename Traits>
> >  void copyToJava(JNIEnv *jni, jobject &target,
> >    const TAO::details::object_reference_const_sequence_element<Traits> &source,
> > -  bool createNewObject = false)
> > +  bool /* createNewObject = false */)
> I think this was supposed to match another usage or something that the idlj2ni compiler generates. We can avoid the warning with bool /*createNewObject*/ = false)
> 
> But if everything works with your change it may be that it's just never called with 3 parameters.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub <https://github.com/objectcomputing/OpenDDS/pull/300/files/094dcd95acb650c544f5c0a592f64d36ff90bfb1#r78228885>, or mute the thread <https://github.com/notifications/unsubscribe-auth/ADwEI6z4mBh5ghvsyhhfcyjrddIkA89lks5qoagOgaJpZM4J5UPs>.
>